desktop/apps/consignments2/client/reducers.js

```diff
@@ -9,29 +9,26 @@ import { routerReducer } from 'react-router-redux'
 const stepsMapping = [
   {
     id: 'create_account',
-    label: 'Create Account',
-    title: ''
+    label: 'Create Account'
   },
   {
     id: 'choose_artist',
-    label: 'Verify Artist/Designer',
-    title: 'Enter the name of the artist/designer who created the work'
```

Comment:
I moved these titles to be inside the components directly, since we were overriding them in half of the cases.